godot-core/src/builtin/string/gstring.rs

```diff
@@ -88,6 +88,10 @@ impl GString {
     ///
     /// Note: This operation is *O*(*n*). Consider using [`chars_unchecked`][Self::chars_unchecked]
     /// if you can make sure the string is a valid UTF-32.
+    #[cfg_attr(
+        since_api = "4.1",
+        deprecated = "Use `chars()` instead.\n Since Godot 4.1 ensures valid UTF-32, checked and unchecked overloads are no longer needed.\nFor details, see [godotengine/godot#74760](https://github.com/godotengine/godot/pull/74760)."
+    )]
```

Comment:
Small semantic clarification:

> Since Godot 4.1 ensures valid UTF-32

This is read as "because Godot 4.1 ensures..."
So I would reword it to:

> Since version 4.1, Godot ensures valid UTF-32...

Reply:
You can also split the string to multiple lines with `\` at the end.

Reply:
A plaine line break without an escape sequence like this works too.
```
#[cfg_attr(
    since_api = "4.1",
    deprecated = "Use `chars()` instead.
    Since version 4.1, ensures valid UTF-32, checked and unchecked overloads are no longer needed.
    For details, see [godotengine/godot#74760](https://github.com/godotengine/godot/pull/74760)."
)]
```
or do you prefere that the lines end with `\n\`?

Reply:
No, this is good 🙂 
If you check the PR docs (auto-generated comment on this PR), it renders as follows:
![deprecation](https://github.com/godot-rust/gdext/assets/708488/89227691-6a6f-4fe7-be2e-539fde1fe5ae)

Reply:
Yeah, I wasn't sure since both render the same but I saw that `\n\` was already used elsewhere.